src/jwtf/src/jwtf.erl

```diff
@@ -112,6 +113,15 @@ validate(Header0, Payload0, Signature, Checks, KS) ->
     Key = key(Header1, Checks, KS),
     verify(Alg, Header0, Payload0, Signature, Key).
 
+validate_checks(Checks) when is_list(Checks) ->
+    ValidChecks = [alg, exp, iat, iss, kid, nbf, sig, typ],
```

Comment:
I would prefer to see these defined at the top of the file along with `ALGS`.

Reply:
definitely.